src/lib/browse/colors.ts

```diff
@@ -4,6 +4,7 @@ export const colors = {
   hospitals: "#B73D25",
   mrn: "#006478",
   parliamentary_constituencies: "#006E59",
+  wards: "purple",
```

Comment:
Back to just making these up. We need a principled way of doing this.

Reply:
I think we're also gonna run out of significantly distinct colours.

Reply:
We could maybe make all the boundary layers one color, and effectively change this to a radio button. But I'm not sure -- we need to know how people want to use this.

Reply:
Yeah, I also imagined we could have a set of colours which got applied to things as you ticked em, and you are only allowed a certain number of layers at once. A changing legend does sound like bad UX though so probably not a good idea